README.md

```diff
@@ -6,21 +6,17 @@
 
 Manages [rbenv][rbenv_site] and its installed Rubies.
 
-Several custom_resources are defined facilitate this.
-
-**WARNING**: As of version 1.0 this cookbook was renamed from rbenv to ruby_rbenv so it could be uploaded to the Supermarket.
-
 ## Requirements
 
 ### Chef
 
-This cookbook requires Chef 12.9+.
+This cookbook requires Chef 13.0+.
```

Comment:
Is there something in these updates that drops support for Chef12?

Reply:
Hey, Chef 12 is about to go EOL, and sous-chefs officially only support non-EOL versions. This just tidies up the test matrix somewhat.